addon/model.js

```diff
@@ -249,6 +263,14 @@ export default class MegamorphicModel extends EmberObject {
     set(this, key, value);
   }
 
+  setProperties(properties) {
+    setProperties(this, properties);
+  }
+
+  addObserver(...args) {
+    addObserver(this, ...args);
+  }
```

Comment:
Are we sure we need this?

If we do to avoid practical breakage then fair enough but i'm surprised at this one

Reply:
We should aim to drop, and not support.

Reply:
@stefanpenner yep that's what i was getting at 👍

Reply:
We had one test using it in M3, happy to refactor that test, was just trying to feel out the surface area of what was getting used.